src/FormBuilder/resources/js/form-builder/src/containers/HeaderContainer.tsx

```diff
@@ -142,6 +156,11 @@ const HeaderContainer = ({
                                     ? __('Save as Draft', 'give')
                                     : __('Switch to Draft', 'give')}
                         </Button>
+                        <Button
+                            label={__('View form', 'give')}
+                            onClick={() => window.location.href = permalink}
+                            icon={external}
+                        />
```

Comment:
Let's make this a proper anchor for a11y:
```suggestion
                        <Button
                            label={__('View form', 'give')}
                            href={permalink}
                            target="_blank"
                            icon={external}
                        />
```